pallets/stateful-storage/Cargo.toml

```diff
@@ -28,12 +28,13 @@ sp-runtime = { workspace = true }
 # Frequency related dependencies
 common-primitives = { default-features = false, path = "../../common/primitives" }
 common-runtime = { path = "../../runtime/common", default-features = false }
+hex = { workspace = true, default-features = false, features = ["alloc"] }
 
 [dev-dependencies]
 env_logger = { workspace = true }
 pretty_assertions = { workspace = true }
 sp-keystore = { workspace = true }
-hex = { workspace = true, default-features = false, features = ["alloc"] }
+pallet-migrations = { workspace = true }
```

Comment:
Q: why is this only a `dev-dependency` if we are using some utilities from it to do multi-block migrations?